gef.py

```diff
@@ -2937,6 +2941,7 @@ def new_objfile_handler(event):
     """GDB event handler for new object file cases."""
     reset_all_caches()
     set_arch()
+    load_libc_args()
```

Comment:
I don't think this needs to be reloaded on every new file

Reply:
i spent some time on this, the only solution i've found is to load definitions from all architectures.
i originally placed load_libc_args() there because it's the first place where current_arch is set and gef knows arch for open binary.

if you're okay for having all json dict loaded at runtime i'll quickly patch my code

Reply:
Ah, thank you for investigating, that actually makes sense. Maybe we can be a bit smart and not re-load if the same arch already has its data loaded.

Reply:
i'm not sure this is really useful, because having a x86-32 loaded you can't load a x86-64 and vice-versa. i have the two messages:
warning: Selected architecture i386 is not compatible with reported target architecture i386:x86-64
and 
warning: Selected architecture i386:x86-64 is not compatible with reported target architecture i386

btw also this has been done, i'm updating the PR

Reply:
What I care more about is someone who has a session and has to wait, let's say a second, every time they reload the file. Have you timed how long this loading takes? If we can load it once that is great.

So what I mean is loading x86 should be skipped if it's already loaded, saving time after the first time.

Reply:
it takes 0.002 seconds on my VM, but i modified the code anyway.
what's unclear to me is why new_objfile_handler() is called 1 time at program load, and then other 3 times as soon as i hit "run".

do you think i should open another issue and read more gef code, so i can try to solve (or at least understand) or this is an expected behaviour?